packages/ui-react/src/containers/Layout/Layout.tsx

```diff
@@ -189,26 +192,27 @@ const Layout = () => {
             <Button key={item.contentId} label={item.label} to={`/p/${item.contentId}`} variant="text" />
           ))}
         </Header>
-        <Sidebar isOpen={sideBarOpen} onClose={() => setSideBarOpen(false)}>
-          <MenuButton label={t('home')} to="/" tabIndex={sideBarOpen ? 0 : -1} />
-          {menu.map((item) => (
-            <MenuButton key={item.contentId} label={item.label} to={`/p/${item.contentId}`} tabIndex={sideBarOpen ? 0 : -1} />
-          ))}
-          <hr className={styles.divider} />
-          {renderUserActions(sideBarOpen)}
-        </Sidebar>
         <div id="content" className={styles.content} tabIndex={-1}>
           <Outlet />
         </div>
+        {!!footerText && (
+          <MarkdownComponent
+            // The extra style below is just to fix the footer on mobile when the dev selector is shown
+            className={classNames(styles.footer, { [styles.testFixMargin]: IS_DEVELOPMENT_BUILD })}
+            markdownString={footerText}
+            tag="footer"
+            inline
+          />
+        )}
       </div>
-      {!!footerText && (
-        <MarkdownComponent
-          // The extra style below is just to fix the footer on mobile when the dev selector is shown
-          className={classNames(styles.footer, { [styles.testFixMargin]: IS_DEVELOPMENT_BUILD })}
-          markdownString={footerText}
-          inline
-        />
-      )}
+      <Sidebar isOpen={sideBarOpen} onClose={() => setSideBarOpen(false)}>
+        <MenuButton label={t('home')} to="/" />
+        {menu.map((item) => (
+          <MenuButton key={item.contentId} label={item.label} to={`/p/${item.contentId}`} />
+        ))}
```

Comment:
The MenuButtons remain focusable via keyboard navigation even when the menu is collapsed. It might be a good idea to adjust the tabIndex on the buttons, ensuring they're skipped in the tab order when the sidebar is not expanded.

```ts
{menu.map((item) => (
  <MenuButton tabIndex={sideBarOpen ? 0 : -1} key={item.contentId} label={item.label} to={`/p/${item.contentId}`} />
))}
```

Reply:
Good find! By applying aria-hidden="true" to the sidebar, I assumed it will be excluded from the accessibility tree, but that was not the case on desktop (at least). I changed this property to inert. This way I do not need to apply tabIndex="-1" to every containing interactieve element. Now it's ignored completely. Thanks for noticing!

Reply:
(It is excluded from the accessibility tree and invisible to screen readers, but doesn't remove them from tab order, or disable keyboard focus)